docs-src/cloud-context/pricing.md

```diff
@@ -0,0 +1,103 @@
+| 120 to 499          | $0.018           |
+| 500 to 1999         | $0.013           |
+| > 2000              | $0.010           |
+
+## Pricing Element: Support
+
+With a subscription to Temporal Cloud you also gain access to our support organization made up of developer success engineers and solution architects. Our team of experts assist with a range of work streams from Workflow design reviews to setting up observability to break/fix support with an agreed-upon set of SLAs.
+
+The Temporal Developer Success team is composed of open source contributors and is backed by the main contributors to the open source Temporal project. They provide deep knowledge of how Temporal works and how you can optimize your deployment.
+
+They also investigate your Workflows to optimize their environments and possibly reduce costs associated with Actions and storage. They ensure your instance is performance tuned and help with other ongoing maintenance, like upgrades of the Temporal software and maintenance of the platform.
+
+If an issue occurs, the team provides support through email, phone, forums, knowledge base and an optional dedicated slack channel. Temporal offers two levels of support defined by their availability and SLAs in the following table.
+
+| **Type**       | **Basic Support**                                                             | **Premium Support**                                                     |
+| -------------- | ----------------------------------------------------------------------------- | ----------------------------------------------------------------------- |
+| Availability   | 0900–1700 Mon–Fri (GMT-9)                                                     | 24×7                                                                    |
+| Response times | P0: 1 business hourP1: 4 business hoursP2: 1 business daysP3: 2 business days | P0: 30 minutesP1: 1 business hourP2: 4 business hoursP3: 1 business day |
+| Pricing        | Greater of $200 or 10% monthly usage (per month)                              | Greater of $2,000 or 10% monthly activity (per month)                   |
+
+## Pricing Element: SSO and SAML
+
+We offer single sign-on (SSO) integration using SAML at a monthly fixed fee based on the number of users registered in Temporal Cloud:
+
+| **Users** | **Cost per Month** |
+| --------- | ------------------ |
+| 0 to 25   | $200               |
+| 26 to 50  | $300               |
+| 51+       | $500               |
+
+## Pricing Estimates
+
+Temporal Cloud employs a consumption-based pricing model that's based on storage and execution, factors that vary from one Workflow to the next. You can estimate the cost of a specific Workflow by running it at a low volume and then using its storage and compute measurements to project your production scale cost. Our team is happy to [help you estimate the cost](https://pages.temporal.io/contact-us) for your specific workloads.
+
+## Temporal Pricing FAQ
```

Comment:
The contents of this section also appear in the proposed Cloud FAQ section (EDU-869). Remove from this page now and link to the FAQ when it's ready? Or remove from FAQ page (which currently leaves only two FAQs)?

```suggestion
## Temporal pricing FAQ
```